<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AAWidget</class>
 <widget class="QWidget" name="AAWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>检测模式 [连续]</string>
   </property>
   <property name="maxVisibleItems">
    <number>15</number>
   </property>
   <item>
    <property name="text">
     <string>检测模式 [连续]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>图形显示 [时序图]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>增益调节 [×1]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>音量调节 [×8]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄色报警阈值[20]dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>红色报警阈值[20]dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>脉冲触发[3]mV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>连续录波[5s]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最大值重置</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{border:1px solid gray;background:transparent;}
QProgressBar::chunk{&quot;background-color:skyblue;&quot;}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_val">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {font-family:WenQuanYi Micro Hei;font-size:52px;color:white}
background:transparent;color:gray;</string>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_max">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true">dBμV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_range">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true">range</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>201</width>
     <height>140</height>
    </rect>
   </property>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>140</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
